shopfloor/services/zone_picking.py

```diff
@@ -429,7 +429,7 @@ def _find_location_move_lines(
         enforce_empty_package=False,
     ):
         """Find lines that potentially need work in given locations."""
-        return self.search_move_line.search_move_lines_by_location(
+        return self.search_move_line.search_move_lines(
             locations or self.zone_location,
             picking_type=picking_type or self.picking_type,
```

Comment:
Already managed by `search_move_lines`
```suggestion
            picking_type=picking_type,
```

Reply:
what about this one ?